client/src/components/ListsDisplay.js

```diff
@@ -47,29 +44,47 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const demoListsArray = [
-  { listTitle: "Clothes", itemCount: 34, img: clothes },
-  { listTitle: "Furniture", itemCount: 12, img: furniture },
-  { listTitle: "Luxury", itemCount: 8, img: luxury },
-];
-
-const ListsDisplay = (props) => {
+const ListsDisplay = () => {
   const classes = useStyles();
-  const { addListOpen, changeAddListOpen } = props;
+  const [lists, setLists] = useState([]);
+  const [userId, setUserId] = useState('-1');
+
+  const getUserId = async () =>{
+    const res = await fetch("/auth/status")
+    const json = await res.json()
+    setUserId(json.data.user_id)
+    console.log(json)
+  }
+
+  const getLists = async () =>{
+    const res = await fetch(`/lists?user_id=${userId}`)
+    const json = await res.json()
+    setLists(json);
+  }
+
+  useEffect(() => {
+    getUserId();
+  }, []);
+
+    useEffect(() => {
+    getLists();
+  }, [userId]);
```

Comment:
Something like this could be combined into a single useEffect that does both once on page load. but it looks like you added getUserId to the userContext, so you can just access it from there rather than making another API call here.